CHANGELOG.md

```diff
@@ -8,11 +8,16 @@ CHANGELOG
 - CentOS 8 is no longer supported (EOL on December 31st, 2021).
 - Upgrade Slurm to version 20.11.8.
 - Upgrade Cinc Client to version 17.2.29.
+- Upgrade NICE DCV to version 2021.2-11190.
+- Upgrade NVIDIA driver to version 470.82.01.
+- Upgrade CUDA library to version 11.4.3.
+- Upgrade NVIDIA Fabric manager to `nvidia-fabricmanager-470`.
```

Comment:
Align the wording to the the other NVIDIA software components: "...to version x.y.z"

Reply:
Done